pkgs/applications/networking/cluster/nomad/default.nix

```diff
@@ -75,11 +75,11 @@ rec {
     passthru.tests.nomad = nixosTests.nomad;
   };
 
-  nomad_1_5 = generic {
+ nomad_1_5 = generic {
```

Comment:
Yup, but you're still dropping the indent here. Once you fix that I'll merge, thanks!

Reply:
:facepalm: Thank you! I totally missed that one. Now I get what you meant with the indent. Fixed.